src/EA.Iws.DataAccess/Repositories/Reports/FinancialGuaranteesRepository.cs

```diff
@@ -33,7 +33,7 @@ public FinancialGuaranteesRepository(IwsContext context)
                       ,[IsBlanketBond]
                       ,[CoverAmount]
                       ,[CalculationContinued]
-                      ,[OverActiveLoads]
+                      ,[OverActiveLoads] = CASE WHEN [OverActiveLoads] = 'Yes' THEN 'Y' WHEN [OverActiveLoads] = 'No' THEN 'N' ELSE 'N' END
```

Comment:
I'm just being really picky here but do you want to just update the `[Reports].[BlanketBonds]` VIEW instead to return Y and N. Then as a result you won't have to change this query in the repository. I can't see the VIEW being used anywhere else.